superset-frontend/src/components/DeleteModal/index.tsx

```diff
@@ -26,7 +26,7 @@ const StyledDiv = styled.div`
   padding-top: 8px;
   width: 50%;
   label {
-    color: ${({ theme }) => theme.colors.grayscale.light1};
+    color: ${({ theme }) => theme.colors.grayscale.dark1};
```

Comment:
```suggestion
    color: ${({ theme }) => theme.colors.grayscale.base};
```
I think now it looks a little TOO dark, and doesn't contrast much with the main text above it. Let's try this `base` value.